python-package/lightgbm/dask.py

```diff
@@ -306,11 +305,11 @@ def _train(
     wait(parts)
 
     for part in parts:
-        if part.status == 'error':
+        if part.status == 'error': # type: ignore
```

Comment:
```suggestion
        if part.status == 'error':  # type: ignore
```

There should be two spaces before every inline comment. This is why the `lint` job in our automated tests is failing.